src/ESPiLight.cpp

```diff
@@ -212,6 +212,10 @@ void ESPiLight::sendPulseTrain(const uint16_t *pulses, int length,
 }
 
 int ESPiLight::send(const String &protocol, const String &json, int repeats) {
+  if (_outputPin < 0) {
+    DebugLn("No output pin set, cannot send");
+    return -1;
+  }
```

Comment:
Good catch!

Reply:
`ESPiLight::sendPulseTrain(const uint16_t *pulses, int length, int repeats)` needs similar.
You forgot to filename in the commit message.

Reply:
`ESPiLight::sendPulseTrain(const uint16_t *pulses, int length, int repeats)` already has it.
```
  if(_outputPin >= 0) {
```

Reply:
I won't consider the commitmessage here as a blocker. Having spend ages in rebasing and cleaning up this might be something forgivable and not worth rebasing everything again.

Who wants to see that file was changed when is free to use `git log --stat`.